UnitsNet/UnitDefinitions/RotationalStiffnessPerLength.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "RotationalStiffnessPerLength",
+  "BaseUnit": "NewtonMeterPerRadianPerMeter",
+  "XmlDoc": "https://en.wikipedia.org/wiki/Stiffness#Rotational_stiffness",
+  "Units": [
+    {
+      "SingularName": "NewtonMeterPerRadianPerMeter",
+      "PluralName": "NewtonMetersPerRadianPerMeter",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "N·m/rad/m", "Nm/rad/m" ]
```

Comment:
Dimensionally, this could be simplified to `N/rad`. I can't find much when googling `RotationalStiffnessPerLength` or `NewtonMeterPerRadianPerMeter`. Are you sure these are widely used units?
I'm a bit hesitant to add quantities or units that are not well documented, that are often very specific to certain applications.

Reply:
@angularsen - All units i need and will be adding in the future are all related to structural engineering.
This particular unit is used when you have a stiffness per length, much like the force per length.
Consider having springs on a surface edge at a certain interval. Instead of having to specify each spring separately in just rotation stiffness, you can specify them in one go using the rotation stiffness per length.
If you want more structural engineering software to use this library, you'll want these units ;)

Reply:
Alright, thanks for the clarification. This absolutely sounds like a widely used domain to me so we can add these, although as discussed in https://github.com/angularsen/UnitsNet/issues/372 proposal 1, we might want to move such quantities out into a separate nugets at some point - although that design comes with some significant challenges we still need to work out